File: PhpFiles/uberV.php
<?php
require_once('uberCar.php');
class uberV extends uberCar 
{
    
    public $typeCarAccepted;
    public $seatsMaterial;
    
    public function __construct($license,$driver,$passengers,$typeCarAccepted, $seatsMaterial)
    {
        parent::__construct($license,$driver,$passengers);
        $this->$typeCarAccepted = $typeCarAccepted;
        $this->$seatsMaterial = $seatsMaterial;
    }
}
?>

File: PythonFiles/uberV.py
from uberCar import uberCar

class uberV:
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license,driver,passengers, typeCarAccepted, seatsMaterial):
        super.__init__(license,driver,passengers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial